extensions/users-permissions/admin/src/components/PopUpForm/index.js

```diff
@@ -0,0 +1,340 @@
+/**
+ *
+ * PopUpForm
```

Comment:
have we forked this from somewhere? it would be good to add a comment with he source and license type

Reply:
No we haven't forked this . Strapi gives us facility to extend the plugin so we just took the code from the Strapi Monorepo and extended it for our use.